docs/content/what-is-a-parent-close-policy.md

```diff
@@ -0,0 +1,37 @@
+---
+id: what-is-a-parent-close-policy
+title: What is a Parent Close Policy
+description: If a Workflow Execution is a Child Workflow Execution, a Parent Close Policy determines what happens to the Workflow Execution should its Parent Workflow Execution change to a Closed status (Completed, Failed, Timed out).
+tags:
+  - explanation
+---
+
+import RelatedReadList from '../components/RelatedReadList.js'
+
+If a Workflow Execution is a Child Workflow Execution, a Parent Close Policy determines what happens to the Workflow Execution should its Parent Workflow Execution change to a Closed status (Completed, Failed, Timed out).
+
+If the Workflow Execution is not a Child Workflow Execution, a Parent Close Policy has no effect on the execution.
+
+A Parent Close Policy must be provided when the Child Workflow Execution is spawned.
+Each Child Workflow Execution may have its own Parent Close Policy.
+
+A Parent Close Policy has three possible values:
```

Comment:
what exactly is the default value if unspecified? i have always wondered and never got a straight answer